calc/src/mechanics/util.ts

```diff
@@ -606,19 +606,21 @@ export function countBoosts(gen: Generation, boosts: StatsTable) {
   return sum;
 }
 
+const formatIV = (iv?: number) => iv !== undefined && iv !== 31 ? ` ${iv} IVs` : '';
+
 export function getEVDescriptionText(
   gen: Generation,
   pokemon: Pokemon,
-  stat: 'atk' | 'def' | 'spd' | 'spa',
-  natureName: NatureName
+  stat: 'atk' | 'def' | 'spd' | 'spa' | 'hp',
```

Comment:
@thejetou I was considering doing this, but speed wasn't included and still isn't, so I hesitated. It does make sense, though.

Reply:
I don't have any real objection to including speed, since it would work, outside of "it's not currently used". 

When we talked about this issue originally on discord, you even mentioned "speed ivs for beast boost", so including this now would cover that feature if/when it does get added.